packages/ui-components/src/components/dropdown-list-item/dropdown-list-item.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
+import { EOtherIconName } from '../icon/icon.types';
+import { IDropdownListItem, IDropdownListItemEvents } from './dropdown-list-item.types';
+
+@Component({
+	tag: 'kv-dropdown-list-item',
+	styleUrl: 'dropdown-list-item.scss',
+	shadow: true
+})
+export class KvDropdownListItem implements IDropdownListItem, IDropdownListItemEvents {
+	/** (required) The text to display on the item */
```

Comment:
Move the documentation blocks to the interfaces on `*.types` so they can be inherent by the classes which implement them.

Then you can leave here just `/** @inheritdoc */`.